gradle/libs.versions.toml

```diff
@@ -29,6 +30,15 @@ spring-boot-three = { id = "org.springframework.boot", version.ref = "springThre
 spring-dependency-management = { id = "io.spring.dependency-management", version = "1.0.11.RELEASE" }
 
 [libraries]
+androidx-activity-compose = { module = "androidx.activity:activity-compose", version = "1.6.3" }
```

Comment:
should we extract the version here since it's repeating a few times below?

Reply:
I avoided that as I wasn't sure if the explicit separate versions were intentional but happy to extract it if there wasn't a reason for that! Mostly just trying to preserve existing behavior.

Reply:
ah, yeah I don't think there was a reason tbh 👍